### PR TITLE
Added ACM Plus information, reorganized Resources page, closes #182

### LINKED ISSUE #182: Add ACM Plus to Resources page

### PR DESCRIPTION
**Pull request by alex-bellon**

### CODE REVIEW

File: resources.html [hunk 1]
```diff
@@ -63,53 +63,63 @@
...
 										<button class="button special" onclick="window.open('http://texasacm.org/forms.html?join')"><span>Sign Up</span></button></center>
 									</div>
-									<div class="6u$ resource">
-										<h2 id="become-officer">Become an Officer</h2>
-										<center><p>Keep an eye out for OO applications in September! For Senior Officer positions, elections are held at the end of every Spring.</p>
-										<button class="button special" onclick="window.open('https://texasacm.org/forms/ooapp')"><span>Apply</span></button></center>
+									<div class="6u$">
+										<h2 id="become-member">Join ACM Plus</h2>
+										<center><p>Get the most out of ACM with <b>ACM Plus membership</b>! For $15 you will receive the latest ACM 2019-2020 t-shirt, have access to all Game Nights for the year, and gain discounts on lockers and soda (with the ability to open a tab for sodas). Join <a href="https://www.texasacm.org/forms/joinPlus" target="_blank">here</a>!</p>
+										<button class="button special" onclick="window.open('http://texasacm.org/forms.html?join')"><span>Sign Up</span></button></center>
```
Comment (rolsanmordua) [inline resources.html]: button links to join, not joinPlus, leads to wrong form

Comment (rolsanmordua) [reply resources.html]: How about making the "Join here" text the button instead of "Sign up"?

Comment (alex-bellon) [reply resources.html]: Done

**Comment by rolsanmordua** [comment]
Removed redundant "Join Here" text, merging.

**Pull request merged**